lib/twitter-ads/account.rb

```diff
@@ -257,6 +256,8 @@ def tailored_audiences(id = nil, opts = {})
     #
     # @since 0.2.3
     def scoped_timeline(id, opts = {})
+      TwitterAds::Utils.deprecated(
```

Comment:
I believe the new Tweets endpoint is ready. So we can either publish this as is (so its in parity with the Python SDK) and make the change in another PR or we can just use the new Tweets endpoint here directly and remove the scoped timeline entirely. I'm fine either way, just wanted to make sure you're aware.

Reply:
Thanks! Yes, I'll be adding the new Tweets endpoint in this PR and let you know again. Though, I'll leave this as is as some of the users might still need some time to migrate.

Reply:
Awesome! Sounds good! Feel free to ping me for a review once you've made the changes

Reply:
@tushdante Added a few commits for the new Tweets endpoint support, thanks